src/WebEid.AspNetCore.Example/WebEid.AspNetCore.Example.csproj

```diff
@@ -11,12 +12,17 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <!-- To switch to .NET 6: Comment out the lines below -->
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="8.0.6" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.6" NoWarn="NU1605" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="8.0.6" NoWarn="NU1605" />
-    <PackageReference Include="Microsoft.Identity.Web" Version="2.19.1" />
-    <PackageReference Include="Microsoft.Identity.Web.UI" Version="2.19.1" />
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.20.1" />
+    <!-- To switch to .NET 6: Uncomment the lines below -->
+    <!--<PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.31" NoWarn="NU1605" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="6.0.31" NoWarn="NU1605" />-->
+
+    <PackageReference Include="Microsoft.Identity.Web" Version="2.20.0" />
+    <PackageReference Include="Microsoft.Identity.Web.UI" Version="2.20.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.21.0" />
```

Comment:
Do you know why we need some VS Azure container tool APIs?

Reply:
I think we do not need them. Removing them did not stop the build from succeeding.
It might be something that Microsoft added to their Asp .Net template, since this has been there since the first version of the project file.

Reply:
Right, got it. Let's remove everything that is not strictly needed when we update authtoken validation library to the new release.

Reply:
I researched a bit more. Initially it seems it is not required, however if the developer wants to use Docker related tools within Visual Studio, then it will not work without it.
Dockerizing with CLI tools works regardless.